src/libs/ValidationUtils.js

```diff
@@ -44,6 +67,31 @@ function isValidDate(date) {
     return moment(date).isValid();
 }
 
+/**
+ * Validates that this is a valid expiration date
+ * in the MM/YY format
```

Comment:
We actually need to send this in the MM/YYYY format. Can we change this to be a function that accepts both MM/YYYY, and MM/YY, but converts MM/YY into MM/YYYY automatically?

Reply:
I think we can modify this validation to accept both MM/YY and MM/YYYY, but leave the conversion to another function that we'll call in the `addBillingCard` API.